# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 339e37a04ca1
# Full Hash: 339e37a04ca17a209114f9d346751e6d05f6a37d
# Author: Lars T Hansen <lhansen@mozilla.com>
# Date: 2021-10-18 16:09:29
# Regressor Bug: 1727084
# File Overlap Count: 1
# Description:
#   Bug 1727084 - Memory64 - Preliminaries. r=yury
#   
#   Scaffolding: Add assertions everywhere execution currently depends on
#   a memory being a memory32.  These assertions will disappear by and by.
#   
# ==============================================================================

diff -r 83e52246d0ea -r 339e37a04ca1 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Mon Oct 18 09:58:13 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Mon Oct 18 09:58:13 2021 +0000
@@ -6208,7 +6208,7 @@
 #if defined(ENABLE_WASM_MEMORY64)
 -   name: javascript.options.wasm_memory64
     type: bool
-    value: true
+    value: false
     mirror: always
 #endif  // defined(ENABLE_WASM_MEMORY64)
 
